src/cartesian/CartesianGrid.js

```diff
@@ -174,7 +254,9 @@ class CartesianGrid extends Component {
 
     return (
       <g className="recharts-cartesian-grid">
-        {this.renderBackground()}
```

Comment:
@costagolub We can use `renderBackground` to render a background rectangle. If you remove this function, we can only achieve this by rendering several rectangles which will bring some strange gaps.

Reply:
@xile611 yes, I actually returned it back in the amended commit.